brigade/core/inventory.py

```diff
@@ -192,12 +192,11 @@ def get_connection(self, connection):
         """
         if connection not in self.connections:
             task_name = "{}_connection".format(connection)
-            try:
-                task = getattr(connections, task_name)
-            except AttributeError:
```

Comment:
why did you remove this?

Reply:
That might have been a merge-timing issue...I might have missed it in the merge conflict. I will re-add.